doc/auth-handlers.md

```diff
@@ -0,0 +1,34 @@
+## AuthImplementations
+
+If we need to enable auth handlers in Pravega controller, modify the pravega manifest and ensure that following fields are present in the yaml file.
+
+```
+authImplementations:
+  mountPath: "/opt/pravega/pluginlib"
+  authHandlers:
+ - image: authHandlerImage
```

Comment:
slight misaligned indent.

Reply:
Fixed , Also verified the upgrade use case. It is working fine